content/python/concepts/operators/operators.md

```diff
@@ -32,47 +35,42 @@ Python divides the operators in the following groups:
 
 ## Arithmetic Operators
 
-Arithmetic operators are used with numeric values to perform common mathematical operations:
+Python has the following arithmetic operators:
 
-| Operator | Name           | Example  |
-| -------- | -------------- | -------- |
-| `+`      | Addition       | `a + b`  |
-| `-`      | Subtraction    | `a - b`  |
-| `*`      | Multiplication | `x * y`  |
-| `/`      | Division       | `x / y`  |
-| `%`      | Modulus        | `x % y`  |
-| `**`     | Exponentiation | `x ** y` |
-| `//`     | Floor division | `x // y` |
+- `+`: Addition, returns the sum of two numbers.
+- `-`: Subtraction, returns the difference of two numbers.
+- `*`: Multiplication, returns the product of two numbers.
+- `/`: Division, returns the quotient of two numbers.
+- `**`: Exponentiation, returns the value of one number raised to the power of another.
+- `%`: Modulus, returns the remainder of one number divided by another.
+- `//`: Floor division, returns integer quotient of two numbers.
```

Comment:
```suggestion
- Addition, `+`, for returning the sum of two numbers.
- Subtraction, `-`, for returning the difference of two numbers.
- Multiplication, `*`, for returning the product of two numbers.
- Division, `/`, for returning the quotient of two numbers.
- Exponents, `**`, for returning the value of one number raised to the power of another.
- Modulus, `%`, for returning the remainder of one number divided by another.
- Floor division, `//`, for returning the integer quotient of two numbers.
```

Reply:
What do you think sounds/reads better? "for returning" or "returns/which returns"?

Reply:
My opinion is to actually keep the symbol operators before the text because it reads clearer.

Also, I think "which returns" sounds better. "`+`: Addition symbol, which returns..."